src/AfriCC/EPP/Client.php

```diff
@@ -329,7 +329,7 @@ private function recv($length)
      * send data to socket
      * @param string $buffer
      */
-    public function send($buffer)
+    private function send($buffer)
```

Comment:
I usually do not like having methods completely private, but it does make sense to change it to protected.